spyder/widgets/status.py

```diff
@@ -254,21 +254,23 @@ def _process_conda_env_info(self):
         if len(parts) >= 2:
             out = ' '.join(parts[:2])
 
-        envs_folder = os.path.sep + 'envs' + os.path.sep
-        if envs_folder in self._interpreter:
-            if os.name == 'nt':
-                env = os.path.dirname(self._interpreter)
+        if 'conda' in self._interpreter:
```

Comment:
I don't know if this is robust enough

Reply:
Do you have a recommendation? I'm happy to improve it, if I can. Is it any less robust than `utils.is_anaconda` or `envs_folder in self._interpreter`?.

Reply:
> Is it any less robust than utils.is_anaconda

Yes it is way less robust that `is_anaconda`, so please fix this

Reply:
> Not sure why you stopped using a function with a clear purpose.

Because it's the wrong function to use here. `utils.is_anaconda` simply checks if `sys.prefix` is a conda environment; this is only Spyder's environment and may or may not be the same environment as `self._interpreter`. But I think I understand your concerns. I've changed it to use `spyder.util.is_conda_env` instead.

Reply:
Yes, I meant that it needed a small refactor to check if it was a conda env.

The same proces that checks for the existence of the file in sys.prefix, applied to the conda env.